docs/project/evolution.md

```diff
@@ -314,9 +314,17 @@ The author of a proposal is not required to include changes to the design
 documentation as part of a proposal, and it may in some cases be preferable to
 decouple the proposal process from updating the design. When accepted, the
 proposal would then be implemented through a series of future PRs to the rest of
-the project, and the proposal document should describe what is being proposed in
-enough detail to validate that those future PRs properly implement the proposed
-direction.
+the project. If the proposal PR defers any documentation changes in this way,
+the proposal document should describe what is being proposed in enough detail to
+validate that those future PRs properly implement the proposed direction, and
+the proposal PR should add "TODO" comments in the places where significant
+future changes will be needed (including adding new placeholder documents if
+needed), with links back to the proposal document. These comments should be kept
+close to the passages that need to be changed. This is intended to ensure that
+readers of the design documentation know when what they're reading is out of
+date, and can easily find out what has changed. See
+[#5606](https://github.com/carbon-language/carbon-lang/pull/5606) for examples
+of adding those comments.
```

Comment:
Had you considered including a brief example in-line? i.e., so people don't need to read this proposal in order to understand what's expected for future proposals. It may also make it easier if we want to evolve in the future, should we want to adjust the format.